lightwood/encoder/categorical/onehot.py

```diff
@@ -3,6 +3,7 @@
 from lightwood.helpers.log import log
 from lightwood.encoder.base import BaseEncoder
 from lightwood.helpers.constants import _UNCOMMON_WORD
+from copy import deepcopy as dc
```

Comment:
Very small nitpick, I think dc is a bit nondescriptive and would suggest using it as deepcopy

Reply:
fixed